src/utils/validator.js

```diff
@@ -10,4 +13,19 @@ export default class Validator {
   static password() {
     return /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])[0-9a-zA-Z]{8,}$/;
   }
+
+  static validateUnbondValue(maxUnbond, textUnbondValue) {
+    try {
+      const unbondValue = parseMerits(textUnbondValue);
+      if (unbondValue.gt(maxUnbond) || unbondValue.lte(BN_ZERO)) return 'Invalid amount';
+      return true;
+    } catch (e) {
+      return 'Invalid amount';
+    }
+  }
+
+  static validateLen(len, text) {
```

Comment:
Why not simply use `minLength` and `maxLength` instead? https://react-hook-form.com/docs/useform/register